frb/surveys/heasarc.py

```diff
@@ -44,14 +48,84 @@ def get_catalog(self):
         self.validate_catalog()
         # Return
         return self.catalog
-        
-        
-class NVSS_Survey(HEASARC_Survey):
+
+
+class SkyView_Survey(surveycoord.SurveyCoord):
```

Comment:
Please add a doc string (and the same for methods, especially
those that are 'new').

Reply:
I've added a few commits for better documentation.